### ISSUE TITLE
Marines are FAR too tanky

### ISSUE THREAD

**Issue by Hmeister-real** [RMC-14/RMC-14#2704]
## Description
As a Hivelord, it takes more than 20+ wideswings to get a SINGLE marine down from full health. It should be far less. 

**Reproduction**

**Screenshots**

**Additional context**
If there are 3 marines per xeno, xenos need to be strong, not stupidly weak if they arent one of the two combat roles

**Comment by diraven** [comment]
Hivelord is not a combat role and doesn't have any combat potential nor it should. Damage handling and all the numbers roughly follow CM13 as far as I know.

**Comment by UnicornOnLSD** [comment]
make every hit in the game, bite, punch, slash, bullet a oneshot, problem fixed :trollface:

**Comment by MACMAN2003** [comment]
> make every hit in the game, bite, punch, slash, bullet a oneshot, problem fixed :trollface:

finally, Realism™

**Comment by DrSmugleaf** [comment]
Yes, this is intended. Combat castes need their abilities to take down marines. Hivelords deal 22.5 Brute damage, the same as drones and the second lowest together with the Boiler, Runner (faster attacks) and Sentinel, all of them second only to the Lesser Drone's 20.
Ravager (the highest) deals 45, and both the Praetorian and Queen deal 40.
The solution to this is to add the remaining combat castes so people don't feel as compelled to play hivelord.
The Warrior (35 melee damage, same as the Lurker) has now been fixed as well, and so will the Crusher be fixed soon as well (40 melee damage), which should help distribute the castes more evently.

https://github.com/RMC-14/RMC-14/assets/10968691/33e72292-0542-4830-a161-136117204fe4
